modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -59,36 +59,37 @@ label:removed[]
 RETURN 1 as my\u0085identifier
 ----
 a|
-The Unicode character \`\u0085` has been removed for unescaped identifiers and is now considered a whitespace character.
-To continue using it, escape the identifier by adding backticks around it.
+The Unicode character `\u0085` (`Next Line`) has been removed for unescaped identifiers and is now considered a whitespace character.
+To continue using it, escape the identifier by adding backticks (++``++) around it.
 This applies to all unescaped identifiers in Cypher, such as label expressions, properties, variable names, or parameters.
-In the given example, the quoted identifier would be \`my�identifier`.
+In the given example, the quoted identifier would be `my\u0085identifier`.
 
 a|
 label:functionality[]
 label:removed[]
 [source, cypher, role="noheader"]
 ----
-RETURN 1 as my$Identifier
+RETURN 1 as my\u0024Identifier
```

Comment:
I think the dollar sign can stay to make it obvious what it is, I wanted the 0085 one changed because it didn't render to anything :)